cmd/zstream/zstream_dump.c

```diff
@@ -38,6 +39,9 @@
 #include <unistd.h>
 #include <stddef.h>
 
+#include <libzfs.h>
+#include <libzfs_core.h>
```

Comment:
Is this "code cleanup" (since something in this header uses stuff from libzfs[_core]), or an inadvertent change?

Reply:
Left over, I'll get rid of it.